roles/openshift_metrics/tasks/install_hawkular.yaml

```diff
@@ -1,50 +1,57 @@
 ---
-- name: generate replication controller templates
+- name: generate hawkular-metrics replication controller
   template:
-    src: "{{ item }}.j2"
-    dest: "{{ mktemp.stdout }}/templates/{{ item }}.yaml"
-  with_items:
-  - hawkular_metrics_rc
-  - hawkular_cassandra_rc
-- name: test view permission for the hawkular service account
-  command: >
-    {{ openshift.common.client_binary }} get rolebindings
-    -o jsonpath='{.items[?(@.roleRef.name == "view")].userNames}'
-  register: ret
-  changed_when: false
-- name: add the view role to the hawkular service account
-  command: >
-    '{{ openshift.common.admin_binary }}' policy
-    -n '{{ project }}' add-role-to-user view hawkular
-  when: "'system:serviceaccount:{{ project }}:hawkular' not in ret.stdout"
-- name: create hawkular metrics objects
-  command: >
-    {{ openshift.common.client_binary }} -n '{{ project }}'
-    create -f '{{ mktemp.stdout }}/templates/{{ item }}'
-  with_items:
-  - hawkular_metrics_secrets.yaml
-  - hawkular_metrics_certificate.yaml
-  - hawkular_metrics_account.yaml
-  - cassandra_secrets.yaml
-  - cassandra_certificate.yaml
-  - metrics-hawkular-sa.yaml
-  - metrics-hawkular-metrics-svc.yaml
-  - hawkular_metrics_rc.yaml
-  - metrics-hawkular-cassandra-svc.yaml
-  - metrics-hawkular-cassandra-nodes-svc.yaml
-  - metrics-cassandra-sa.yaml
-- name: process cassandra template
-  shell: >
-    {{ openshift.common.client_binary }} -n '{{ project }}'
-    process -f '{{ mktemp.stdout }}/templates/hawkular_cassandra_rc.yaml'
-    --value 'NODE={{ item }}'
-    {% if item == '1' %}--value MASTER=true{% endif %}
-    | oc create -f -
-  with_sequence: start=1 end={{ hawkular_cassandra_nodes }}
-- name: create the hawkular-metrics route
-  command: >
-    {{ openshift.common.client_binary }} -n '{{ project }}'
-    create route reencrypt hawkular-metrics
-    --hostname='{{ hawkular_metrics_hostname }}'
-    --service=hawkular-metrics
-    --dest-ca-cert='{{ mktemp.stdout }}/certs/ca.crt'
+    src: hawkular_metrics_rc.j2
+    dest: "{{ mktemp.stdout }}/templates/hawkular_metrics_rc.yaml"
+- name: generate hawkular-cassandra replication controllers
```

Comment:
i agree with that, but I didn't think they have a random name as part of any of their components as we do for ES